Removed some unnecessary preset parameters for FW

b14ckyy opened the pull request:
Removed some Preset values for Fixed wings due to FW changes that will be good on defaults.
Comes along with https://github.com/iNavFlight/inav/pull/10541

mmosca commented:
Maybe instead of removing these, just adjust the airplane without a tail values as bredoven recommendations? We can look into removing this later, with more time for testing.

b14ckyy commented:
If the controller now controls velocity and not position, I think the difference in wing load and pitch authority should not make a difference between them anymore. What is breadoven's recommendation? Could not see that in the 2 threads I follow. If they still act differently then yes I can add them back in with slightly different values.

breadoven commented:
> If the controller now controls velocity and not position, I think the difference in wing load and pitch authority should not make a difference between them anymore. What is breadoven's recommendation? Could not see that in the 2 threads I follow. If they still act differently then yes I can add them back in with slightly different values.

I don't see any reason why the velocity altitude control would affect the xy position control so these values should be left unchanged.

b14ckyy commented:
@breadoven I removed the xy presets in that go as well since I noticed that somewhere in the 7.0 or 7.1 phase the value for flying wings with no tail was too low. I planned to change/remove them for a while and just remembered so I removed them now. 
Especially I noticed with Path tracking and Autoland, that Wings struggle to get a precise heading fast enough on 55 while the Tail Planes worked much better. 75 or more on wings was severely better in tracking.

b14ckyy commented:
I have restored the settings and just changed the defaults slightly based on the recommendations.

mmosca commented:
This missed the train for 8.0. What is the concensus? Is it still needed?

b14ckyy commented:
Not mandatory
